knowledge-base/pdf-invisible-signatures.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to Create Invisible Signatures for PDF Documents
+description: Learn how to digitally sign a PDF document using RadPdfProcessing for Document Processing.
+type: how-to
+page_title: How to Create Invisible Signatures for PDF Documents
+slug: pdf-invisible-signatures
+tags: pdf, signing, digital signature, pdfprocessing, invisible 
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.124 | RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+This article shows how to sign PDF documents with invisible signatures. 
+
+## Solution
+
+The invisible signature is created by not setting the size to the signature widget. It can not be achieved without the use of signature flags.
```

Comment:
"It can not be achieved without..." - Maybe turn this into a positive: "This can be achieved with the use of signature flags..."